ddtrace/vendor/contextvars/__init__.py

```diff
@@ -0,0 +1,160 @@
+import threading
```

Comment:
do we need to update `vendor/__init__.py` as well?

Reply:
looks like it!